core/src/main/scala/org/ensime/core/RichPresentationCompiler.scala

```diff
@@ -478,32 +478,14 @@ class RichPresentationCompiler(
           }
         case Annotated(atp, _) =>
           List(atp.symbol)
-        case st: SymTree =>
+        case ap @ Select(qualifier, nme.apply) =>
```

Comment:
This is taken from [scala-ide ](https://github.com/scala-ide/scala-ide/blob/b2cb96d3d6735a4eb243a35b09e7835c8f8f8e18/org.scala-ide.sdt.core/src/org/scalaide/core/internal/hyperlink/ScalaDeclarationHyperlinkComputer.scala#L53)